site/content/en/docs/tasks/manage/enforce_job_management/setup_default_local_queue.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Setup default LocalQueue"
+date: 2024-12-12
+weight: 10
+description: >
+  Setup default LocalQueue to fullfil a queue label on jobs that submited without queue label.
+---
+
+This page describes how to setup default LocalQueue to ensure that all Workloads submitted to a specific namespace, intended for Kueue users, are managed by Kueue, even if the `kueue.x-k8s.io/queue-name` label is not specified explicitly. 
```

Comment:
```suggestion
This page describes how to setup default LocalQueue to ensure that all Workloads submitted to a specific namespace are managed by Kueue, even if the `kueue.x-k8s.io/queue-name` label is not specified explicitly. 
```
maybe it is not that relevent that it is "indended for Kueue users". And dropping this part makes it more readable imo.